devices/tuya/_TZ3000_yd2e749y_temp_hum_sensor.json

```diff
@@ -1,6 +1,6 @@
 {
   "schema": "devcap1.schema.json",
-  "manufacturername": ["_TZ3000_yd2e749y", "_TZ3000_fllyghyj"],
+  "manufacturername": ["_TZ3000_yd2e749y", "_TZ3000_fllyghyj", "_TZ3000_fie1dpkm"],
   "modelid": ["TS0201", "TS0201"],
```

Comment:
Hi the "modelid" array, needs to have the same size as the "manufacturername" array, can you please add the entries matching the device (likely also TS0201).

Reply:
Sure, consider it done. Makes sense, just missed it.